recipe/meta.yaml

```diff
@@ -1,64 +1,106 @@
 # version numbering is explained at https://stackoverflow.com/questions/42730478/version-numbers-for-visual-studio-2017-boost-and-cmake
-#     This version number follows the "platform toolset"
+# This version number follows the "platform toolset"
+
 {% set vcver = "14.1" %}
-{% set vsyear = "2017" %}
-{% set fullver = runtime_version %}
-# VS2017 is fundamentally compatible with VS2015.  We name our package vs2015_runtime so that it can't be
-#    mixed up with the runtime from VS2015 - you do actually need VS2017 runtime for things compiled with
-#    VS2017, and the runtime we have here is backwards-compatible with things compiled with VS2015.
+{% set vsyear = "2019" %}
+{% set vsver = "16" %}
+{% set runtime_version = "14.26.28720" %}
+
+# VS2017 is fundamentally compatible with VS2015.  We name our package
+# vs2015_runtime so that it can't be mixed up with the runtime from
+# VS2015 - you do actually need VS2017 runtime for things compiled
+# with VS2017, and the runtime we have here is backwards-compatible
+# with things compiled with VS2015.
 {% set runtime_year = "2015" %}
 
+# the VS update version.  This is the middle digit in the version
+# reported in the VS help->about UI.  It is perhaps a more readily
+# referenceable number.
+{% set update_version = "6" %}
+
+#  This is the version number reported by cl.exe
+{% set cl_version = "19.26.28806" %}
+
 package:
   name: vs{{ vsyear }}
-  version: {{ fullver }}
+  version: {{ runtime_version }}
+
+source:
+  - url: https://aka.ms/vs/16/release/vc_redist.x64.exe  # [win64]
```

Comment:
Is this a stable link?

Reply:
Yes, that is the stable link which redirects to their CDN.

Reply:
I mean, they update the actual file from time to time right? The file I get today might not be the same one I'll get in a few weeks right?